### PR TITLE
Rollup script: Log sizes and temporarily disable one exception

### PR DESCRIPTION
**Pull request by rshipp**
Followup with @pablo-code-org's suggested approach from #51849.

Add logging for all three object sizes, and temporarily comment out the first exception that we know is raising.

Also fixes a typo in the second exception message.

## Links

- [slack thread on this approach](https://codedotorg.slack.com/archives/C03CK49G9/p1683665914631939)
- jira ticket: [TEACH-392](https://codedotorg.atlassian.net/browse/TEACH-392)

## Followup task

Send some of these queries to a replica database, to reduce the load on production.

jira: [TEACH-501](https://codedotorg.atlassian.net/browse/TEACH-501)

### CODE REVIEW

**Review by pablo-code-org — APPROVED** [review]
Thank you! This should be good to go

**Review by davidsbailey — APPROVED** [review]
seems like a reasonable step given the relatively complex query that's needed to compute the number of contained levels, although from context it does seem very likely to be just over the previous limit.

File: bin/cron/create_rollup_tables [hunk 1]
```diff
@@ -144,8 +144,12 @@ def main
   end
 
   # Verify that the data size is sensible.
-  raise "contained_levels too big: #{contained_levels.size} rows exceeds limit of 10000" if contained_levels.size > 10_000
-  raise "contained_level_answers too big: #{contained_levels_answers.size} rows exceeds limit of 100000" if contained_level_answers.size > 100_000
+  CDO.log.info "contained_levels: #{contained_levels.size}"
+  CDO.log.info "contained_level_answers: #{contained_level_answers.size}"
+  CDO.log.info "level_sources: #{level_sources_multi_types.size}"
```
Comment (bethanyaconnor) [inline bin/cron/create_rollup_tables:149]: Just for my knowledge -- and I should know this -- where can we see these logs?

**Review by bethanyaconnor — APPROVED** [review]
LGTM!

File: bin/cron/create_rollup_tables (hunk 1 above)
Comment (rshipp) [reply bin/cron/create_rollup_tables:149]: I will also need to figure this out 😅

Comment (rshipp) [reply bin/cron/create_rollup_tables:149]: they're in cloudwatch log groups: https://codedotorg.slack.com/archives/C0T0PNTM3/p1684520216970339

**Pull request merged**